pytab/core.py

```diff
@@ -361,6 +362,13 @@ def cli(
     click.echo("| Obtaining System Information...", nl=False)
     system_info = hwi.get_system_info()
     click.echo(" success!")
+    click.echo("| Detected System Config:")
+    click.echo(yaml.dump(system_info, default_flow_style=False))
```

Comment:
Especially on systems with multiple RAM Sticks, GPUs, CPUs, ... this will be absolutely uchaotic.
There is no way that a user will read through this and see, what device has what specs and so on...

This way of outputing is useless in my oppinion.